helm-charts/basehub/values.yaml

```diff
@@ -445,10 +445,33 @@ jupyterhub:
         args:
           - -c
           - |
-            ls -lhd /srv/repo;
-            while true; do git fetch origin;
-            git reset --hard origin/$(GIT_REPO_BRANCH);
-            sleep 5m; done
+            handle_sigterm() {
+                echo "SIGTERM received, terminating...";
+                exit;
+            }
+            trap handle_sigterm SIGTERM
+
+            echo "Starting template sync...";
+
+            # print debugging info about commands run and their output using a
+            # temporary subshell with "set -x"
```

Comment:
Can you add a comment on *why* this is done rather than just using `set -x` globally for the whole script?